apps/web/pages/api/book/event.ts

```diff
@@ -290,6 +291,46 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
     return g;
   });
 
+  // For seats, if the booking already exists then we want to add the new attendee to the existing booking
+  if (reqBody.bookingId) {
+    if (!eventType.seatsPerTimeSlot)
+      return res.status(404).json({ message: "Event type does not have seats" });
+
+    const booking = await prisma.booking.findUnique({
+      where: {
+        id: reqBody.bookingId,
+      },
+      include: {
+        attendees: true,
+      },
+    });
+    if (!booking) return res.status(404).json({ message: "Booking not found" });
+
+    if (eventType.seatsPerTimeSlot <= booking.attendees.length)
+      return res.status(409).json({ message: "Booking seats are full" });
+
+    if (booking.attendees.some((attendee) => attendee.email === invitee[0].email))
+      return res.status(409).json({ message: "Already signed up for time slot" });
+
+    await prisma.booking.update({
+      where: {
+        id: reqBody.bookingId,
+      },
+      data: {
+        attendees: {
+          create: {
+            email: invitee[0].email,
+            name: invitee[0].name,
+            timeZone: invitee[0].timeZone,
+            locale: invitee[0].language.locale,
```

Comment:
We are assuming we are only creating one attendee since we are disabling guests for event types with seats